### PR TITLE
test: Better repository name generation

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]

File: src/sentry/testutils/fixtures.py [hunk 1]
```diff
@@ -367,7 +367,8 @@ def create_release(self, project, user=None, version=None):
     def create_repo(self, project, name=None):
         repo = Repository.objects.create(
             organization_id=project.organization_id,
-            name=name or make_word(),
+            name=name or '{}-{}'.format(petname.Generate(2, '',
+                                                         letters=10) - random.randint(1000, 9999)),
```
Comment (alex-hofsteede) [inline src/sentry/testutils/fixtures.py]: `unsupported operand type(s) for -: 'str' and 'int'`

**Review by alex-hofsteede — CHANGES_REQUESTED** [review]
seems like the subtraction is a bug

**Comment by dcramer** [comment]
@alex-hofsteede yeah fwiw i was relying on Ci to tell me if the code ran or not ;)

**Comment by dcramer** [comment]
gucci disappoint

**Review by alex-hofsteede — APPROVED** [review]

**Pull request merged**